$NetBSD: patch-aa,v 1.5 2013/04/24 06:45:32 sbd Exp $

- fix a typo about LDFLAGS
- fix paths, some of them being hardcoded.

--- configure.orig	2007-02-28 22:33:11.000000000 +0000
+++ configure
@@ -2957,7 +2957,7 @@ fi
 echo "$as_me: libdir is $libdir" >&6;}
 #filterdir=/usr/lib/cups/filter
 #AC_SUBST(filterdir)
-KM_PPDDIR=/usr/share/cups/model/KONICA_MINOLTA
+KM_PPDDIR=${datadir}/cups/model/KONICA_MINOLTA
 
 
 UNAME=`(uname) 2>/dev/null || echo unknown`
@@ -2971,7 +2971,7 @@ cat >>confdefs.h <<_ACEOF
 #define CUPS_SERVERBIN "$exec_prefix/libexec/cups"
 _ACEOF
 
-		LDFLAGS="$LDFALGS -L/usr/local/lib"
+		LDFLAGS="$LDFLAGS -L/usr/local/lib"
 		CPPFLAGS="$CPPFLAGS -I/usr/local/include -I/sw/include"
 		KM_DATADIR=/Library/Printers/KONICA_MINOLTA/mc5440DL
 
@@ -2980,7 +2980,7 @@ cat >>confdefs.h <<_ACEOF
 _ACEOF
 
                 ;;
-        *BSD*)
+        *BSD* | DragonFly*)
                 # *BSD
                 CUPS_SERVERBIN="$exec_prefix/libexec/cups"
 
@@ -2988,7 +2988,7 @@ cat >>confdefs.h <<_ACEOF
 #define CUPS_SERVERBIN "$exec_prefix/libexec/cups"
 _ACEOF
 
-		LDFLAGS="$LDFALGS -L/usr/local/lib"
+		LDFLAGS="$LDFLAGS -L/usr/local/lib"
 		CPPFLAGS="$CPPFLAGS -I/usr/local/include"
 		KM_DATADIR="$datadir/KONICA_MINOLTA/mc5440DL"
 
@@ -2999,10 +2999,10 @@ _ACEOF
                 ;;
         *)
                 # All others
-                CUPS_SERVERBIN="$libdir/cups"
+                CUPS_SERVERBIN="$exec_prefix/libexec/cups"
 
 cat >>confdefs.h <<_ACEOF
-#define CUPS_SERVERBIN "$libdir/cups"
+#define CUPS_SERVERBIN "$exec_prefix/libexec/cups"
 _ACEOF
 
 		KM_DATADIR="$datadir/KONICA_MINOLTA/mc5440DL"
@@ -3239,7 +3239,7 @@ elif test -f "/usr/lib/libjbig.a"; then
 elif test -f "/usr/local/lib64/libjbig.a"; then
 	LIBJBIG="/usr/local/lib64/libjbig.a"
 else
-	LIBJBIG="/usr/local/lib/libjbig.a"
+	LIBJBIG="${libdir}/libjbig.a"
 fi
 else
   { { echo "$as_me:$LINENO: error: Could not find libjbig.a" >&5
@@ -3320,7 +3320,7 @@ elif test -f "/usr/lib/liblcms.a"; then
 elif test -f "/usr/local/lib64/liblcms.a"; then
 	LIBLCMS="/usr/local/lib64/liblcms.a"
 else
-	LIBLCMS="/usr/local/lib/liblcms.a"
+	LIBLCMS="${libdir}/liblcms.a"
 fi
 else
   { { echo "$as_me:$LINENO: error: Could not find liblcms.a" >&5
